--- a/homeassistant/components/matter/config_flow.py
+++ b/homeassistant/components/matter/config_flow.py
@@ -35,7 +35,7 @@
 
 ADDON_SETUP_TIMEOUT = 5
 ADDON_SETUP_TIMEOUT_ROUNDS = 40
-DEFAULT_URL = "ws://localhost:5580/ws"
+DEFAULT_URL = "unix:///var/snap/home-assistant/current/matter.socket"
 DEFAULT_TITLE = "Matter"
 ON_SUPERVISOR_SCHEMA = vol.Schema({vol.Optional(CONF_USE_ADDON, default=True): bool})
 
